test/testUtils.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+const {sync: spawnSync} = require('execa');
+
+const WEBPACK_PATH = path.resolve(__dirname, "../bin/cli.js");
+
+function runWebpack(
+	testCase,
+	args,
+) {
+	const cwd = path.resolve(__dirname, 'binCases', testCase);
+	const outputPath = path.resolve(__dirname, 'js/bin', testCase)
+	const argsWithOutput = args.concat('--output-path', outputPath)
+
+	const result = spawnSync(WEBPACK_PATH, argsWithOutput, {
+		cwd,
+		reject: false,
+	});
+
+	result.stdout = result.stdout === '' ? [] : result.stdout.split('\n');
```

Comment:
One thing I would do instead of making this into an array would be to normalize the output (strip out webpack version, remove timestamps, and maybe bundle sizes), then snapshot the output. Up to you, though 🙂